packages/image_picker/image_picker_platform_interface/lib/src/types/image_picker_options.dart

```diff
@@ -0,0 +1,46 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:image_picker_platform_interface/src/types/types.dart';
+
+/// Specifies options for picking a single image from the device's camera or gallery.
+class ImagePickerOptions {
+  /// Creates an instance with the given [maxHeight], [maxWidth], [imageQuality],
+  /// [referredCameraDevice] and [requestFullMetadata].
+  const ImagePickerOptions({
+    this.maxHeight,
+    this.maxWidth,
+    this.imageQuality,
+    this.preferredCameraDevice = CameraDevice.rear,
+    this.requestFullMetadata = true,
+  });
+
+  /// The maximum width of the image, in pixels. If null, the image will only
+  /// be resized if maxHeight is specified.
```

Comment:
Separate additional description and the summary with 1 empty line: 
```suggestion
  /// The maximum width of the image, in pixels.
  ///
  /// If null, the image will only be resized if [maxHeight] is specified.
```

ditto below